.github/workflows/python-deps.yml

```diff
@@ -27,11 +27,23 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Install prerequisites
+        # brew can be finicky but it does not always means that the rest of the job will fail
+        run: |
+          # Needed for xmlsec
+          brew install libxml2 libxmlsec1 pkg-config
+
       - name: Setup Python
         uses: ./.github/actions/setup-python
+        with:
+          # Non-default value since install-py-dev installs all requirements files
+          # We also want to bust the cache if the action and workflow change
+          cache-files-hash: ${{ hashFiles('requirements-*.txt', '.github/actions/setup-python/action.yml', '.github/workflows/python-deps.yml') }}
 
       - name: Install dependencies
         run: |
           python -m venv .venv
           source .venv/bin/activate
           make install-py-dev
+          # This exercises the requirements-pre-commit.txt file
```

Comment:
This is just because there's no coverage for macos in backend lint, right?

Reply:
As you hint, we don't execute it on Macs.